lib/presentation/atoms/list_tile_atom.dart

```diff
@@ -21,19 +24,46 @@ class ListTileAtom extends StatelessWidget {
   final bool translateSubtitle;
   final VoidCallback? onTap;
   final bool enable;
+  final ListTileVariant variant;
+  final bool isCrossed;
 
   @override
   Widget build(BuildContext context) {
-    return ListTile(
-      enabled: enable,
-      contentPadding: EdgeInsets.zero,
-      title: TextAtom(title, translate: translateTitle),
-      subtitle: subtitle == null
-          ? null
-          : TextAtom(subtitle!, translate: translateSubtitle),
-      leading: leading,
-      trailing: trailing,
-      onTap: onTap,
-    );
+    switch (variant) {
+      case ListTileVariant.regular:
+        return ListTile(
+          enabled: enable,
+          contentPadding: EdgeInsets.zero,
+          title: TextAtom(title, translate: translateTitle),
+          subtitle: subtitle == null
+              ? null
+              : TextAtom(subtitle!, translate: translateSubtitle),
+          leading: leading,
+          trailing: trailing,
+          onTap: onTap,
+        );
+      case ListTileVariant.strikethrough:
+        return ListTile(
+          enabled: enable,
+          contentPadding: EdgeInsets.zero,
+          title: AnimatedLineThrough(
```

Comment:
Please move AnimatedLineThrough to a separate atom, we will implement the sound effect in the future inside of the atoms